fastly/base_fastly_service_v1.go

```diff
@@ -340,13 +340,14 @@ func resourceServiceUpdate(ctx context.Context, d *schema.ResourceData, meta int
 	initialVersion := false
 
 	if needsChange {
-		latestVersion := d.Get("active_version").(int)
-		if latestVersion == 0 {
+		var latestVersion int
+		if isCreate {
```

Comment:
I don't think we need the `if` block now. Could we not just now rely on the `else` block which does...

```
latestVersion = d.Get("cloned_version").(int)
```

Because inside of `resourceServiceCreate` we now set `cloned_version`.

Reply:
I thought this, but if we do then we end up cloning immediately after creation, if that makes sense? So on a normal creation we'd go: "create service, set cloned_version=1, call Update function, apply comment etc to version 1, clone, set cloned_version=2, make updates to nested blocks" all in a single `apply` and end up with two versions straight away

Reply:
Ok that makes sense. Thanks!